Lib/glyphsLib/builder/glyph.py

```diff
@@ -126,6 +126,26 @@ def to_ufo_glyph(self, ufo_glyph, layer, glyph):
     else:
         ufo_glyph.width = width
 
+    if self.is_vertical:
+        if "typoAscender" in layer.master.customParameters:
+            ascender = layer.master.customParameters["typoAscender"]
+        else:
+            ascender = layer.master.ascender
```

Comment:
I checked Glyphsapp and if the typoAscender or typoDescender are missing, glyphsApp will use the master.ascender and master.descender instead. It doesn't use the vhea customParameters.